rest/route/host_create.go

```diff
@@ -157,18 +160,18 @@ func makeContainerLogsRouteManager(sc data.Connector) gimlet.RouteHandler {
 func (h *containerLogsHandler) Factory() gimlet.RouteHandler { return &containerLogsHandler{sc: h.sc} }
 
 func (h *containerLogsHandler) Parse(ctx context.Context, r *http.Request) error {
-	id := gimlet.GetVars(r)["container_id"]
-	host, err := host.FindOne(host.ById(id))
+	id := gimlet.GetVars(r)["evergreen_id"]
```

Comment:
Instead of evergreen_id let's use host_id, since that's consistent with how it's named elsewhere